connectors/src/connectors/slack/chat/stream_conversation_handler.ts

```diff
@@ -131,6 +131,11 @@ export async function streamConversationToSlack(
   let answer = botIdentity;
   const actions: AgentActionType[] = [];
   for await (const event of streamRes.value.eventStream) {
+    // This logger.info(Received event from Dust) is to be removed after august 30 2024.
+    logger.info(
+      { connectorId: connector.id, conversationId: conversation.sId },
+      "Received event from Dust"
```

Comment:
Should we add the event.type?

Reply:
ouch, added it initially, but wrong copy/paste got rid of it.

Reply:
Updated @flvndvd